pkg/healthcheck/mock_test.go

```diff
@@ -64,6 +64,10 @@ func newGRPCServer(healthSequence ...healthpb.HealthCheckResponse_ServingStatus)
 	return gRPCService
 }
 
+func (s *GRPCServer) List(_ context.Context, _ *healthpb.HealthListRequest) (*healthpb.HealthListResponse, error) {
+	return nil, nil
+}
+
```

Comment:
Is it just because you stumbled onto this, or is it somehow a hard requirement to fix this now? I don't understand why there is this change TBH